src/utils/dispute-utils.js

```diff
@@ -216,30 +225,36 @@ export function getPhaseAndTransition(dispute, courtConfig, date) {
     let currentAdjudicationPhase = getAdjudicationPhase(
       dispute,
       lastRound,
-      date,
+      currentTerm,
       courtConfig
     )
-    if (currentAdjudicationPhase.phase === DisputesTypes.Phase.Ended) {
+
+    const { phase, phaseEndTerm } = currentAdjudicationPhase
+    if (phase === DisputesTypes.Phase.Ended) {
       currentAdjudicationPhase = {
         ...currentAdjudicationPhase,
         phase: DisputesTypes.Phase.ExecuteRuling,
       }
     }
-    return { ...currentAdjudicationPhase, roundId: number }
+
+    const nextTransition = phaseEndTerm
```

Comment:
Since we weren't passing this back before, I'm wondering what this will do

Reply:
Ah actually we were passing this, just that we changed how we compute it. 

Before we were returning it along the phase at `getAdjudicationPhase`, (see for example https://github.com/aragon/court-dashboard/pull/338/files#diff-0311c4b36f556c7a49619bd3afa3d955L263) 

Now instead, we return the term at which the phase ends and compute the `endTime` of said term. This change was mainly to prevent computing the `nextTransition` on every return in `getAdjudicationPhase` which required calling `getTermEndTime` every time